README.md

```diff
@@ -1,2 +1,60 @@
 # monthly-challenges
 Repository containing monthly challenges about quantum computing.
+
+## How it works
+
+These challenges are meant to keep you active and prepared until the next round of the mentorship challenge.  
```

Comment:
TBH I would rather say that they're meant to help you hone your QC skills. Mentorship stuff is secondary in principle ;)

Reply:
Fair enough, I'll change that.